detox/src/android/expect.js

```diff
@@ -1,5 +1,6 @@
 const invoke = require('../invoke');
```

Comment:
Don't you want to add the other actions as well ?
Did you run the Android E2E suite with the new implementation ?

Reply:
I ran the e2e suite with the old implementation, will re-run it the next days with the newer one. As I said, I would like to add the actions step by step to have smaller, easier to merge PRs.